ruby/vampires.rb

```diff
@@ -110,3 +110,5 @@
 end
```

Comment:
For Line 26:

When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`